lib/features.json

```diff
@@ -318,5 +318,10 @@
     "name": "setSlotAsTransaction",
     "description": "use setslot as transaction instead of just hoping it'll work",
     "versions": ["1.17", "1.17.1"]
+  },
+  {
+    "name": "armAnimationBefore",
```

Comment:
A bit off an nitpick but maybe change armAnimationBefore to maybe `armAnimationBeforeUse` to make it more descriptive.

Reply:
Will do 👍